hawkular-inventory-impl-tinkerpop-parent/hawkular-inventory-impl-tinkerpop/src/main/java/org/hawkular/inventory/impl/tinkerpop/TinkerpopInventory.java

```diff
@@ -74,7 +75,7 @@
                         .withProperty(IndexSpec.Property.builder()
                                 .withName(Constants.Property.__eid.name())
                                 .withType(String.class)
-                                .withUnique(true)
+//                                .withUnique(true)
```

Comment:
will this work? What if I create the entity which already exists? Does the `.withUnique(true)` on line 117 take care of that?